cli/example.yaml

```diff
@@ -19,8 +19,20 @@ maturity: MATURITY_BETA
 ## Copyright holder(s) of the generated files and classes.
 copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
 
-auth_features:
+atto_features:
+    ## Adds strings to be initialized in the plugin's user interface.
+    strings_for_js:
```

Comment:
In fact this is not Atto specific feature. Any plugin of any type that makes use of YUI (or actually any JS) could use it. On the other hand, I do understand that this way of defining strings has been de facto deprecated everywhere but in Atto that is whole built around YUI.